gym-unity/gym_unity/envs/__init__.py

```diff
@@ -344,6 +358,79 @@ def _check_agents(self, n_agents: int) -> None:
                 "initialization. This is not supported."
             )
 
+    def _sanitize_info(self, info: BatchedStepResult) -> BatchedStepResult:
+        self._stored_info = info  # store the new original
+        if info.n_agents() == self._n_agents:
+            return info
+        n_extra_agents = info.n_agents() - self._n_agents
+        if self._n_agents is None:
```

Comment:
It seems like either this is never true, or the math on the line above is bad (can't do `int` - `None`)

Reply:
I think this should never be true as `self.n_agents` will be assigned by the `check_agents` call, which is done before this method

Reply:
I got mypy freeking out

Reply:
I'll take a look